cmd/manager/server.go

```diff
@@ -103,35 +99,31 @@ func run() {
 		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
 	})
 	if err != nil {
-		log.Errora(err, "")
-		os.Exit(1)
+		log.Fatalf("Could not create a controller manager: %v", err)
 	}
 
-	log.Info("Registering Components.")
+	log.Infof("Registering Components.")
```

Comment:
no format specified here, keep it as Info?

Reply:
will do.